cluster/versions.tf

```
@@ -2,11 +2,11 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 5.3"
+      version = ">= 6.17"
```

I guess, this should be
```suggestion
      version = ">= 5.3"
```